lib/lib/date.ts

```diff
@@ -0,0 +1,13 @@
+import { format } from "date-fns"
+
+export function formatTime(date: Date) {
+  return format(date, "HH:mm")
+}
```

Comment:
We should use `p` which is a localized version of the time to use 12h or 24h version when necessary

<img width="525" alt="Screenshot 2024-12-10 at 11 01 08" src="https://github.com/user-attachments/assets/25dcb92c-3011-455b-a003-9a1540651657">